opendebates/tests/test_registration.py

```diff
@@ -131,4 +131,4 @@ def test_logout(self):
         rsp = self.client.get(logout_url)
         self.assertRedirects(rsp, self.home_url)
         rsp = self.client.get(self.home_url)
-        self.assertIn('Log in', rsp.content)
+        self.assertIn('Log in', rsp.content)        
```

Comment:
Ahhh!  Wow, good catch.

I think I've fixed these now.

On Tue, Apr 5, 2016 at 11:59 AM, Vinod Kurup <notifications@github.com>
wrote:

> In opendebates/tests/test_registration.py
> <https://github.com/caktus/django-opendebates/pull/106#discussion_r58565273>
> :
>
> > @@ -131,4 +131,4 @@ def test_logout(self):
> >          rsp = self.client.get(logout_url)
> >          self.assertRedirects(rsp, self.home_url)
> >          rsp = self.client.get(self.home_url)
> > -        self.assertIn('Log in', rsp.content)
> > +        self.assertIn('Log in', rsp.content)
>
> Also... trailing whitespace here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/caktus/django-opendebates/pull/106/files/93bf64480851bd229a70c26d8140ef5e16067970#r58565273>
>